backend/LoggingBenchmark.WebApp/LoggingExtensions.cs

```diff
@@ -0,0 +1,58 @@
+using Elastic.Extensions.Logging;
+using OpenTelemetry;
+using OpenTelemetry.Logs;
+
+namespace LoggingBenchmark.WebApp;
+
+public static class LoggingExtensions
+{
+    public static void AddLogging(this WebApplicationBuilder builder)
+    {
+        builder.Logging.ClearProviders();
+
+        var benchmarkType = builder.Configuration.GetValue<BenchmarkType>("BenchmarkType");
+
+        switch (benchmarkType)
+        {
+            case BenchmarkType.OtelConsole:
+                builder.AddOtelConsoleLogging();
+                break;
+            case BenchmarkType.ElasticsearchHttpClient:
+                builder.AddElasticsearchHttpClientLogging();
+                break;
+        }
+
+        builder.UseOtlpExporterIfAspire();
```

Comment:
I think it is important to make it as clear as possible because we want to explore performance. For example, we don't need it when the app uses `Elasticsearch`, but we need `OtlpExporter` for `Aspire`.

But I may miss something important, what do you think?